simpl/games/apps.py

```diff
@@ -1,7 +1,7 @@
 from django.apps import AppConfig
 from django.contrib.auth import get_user_model
 
-from thorn import model_reverser, webhook_model
+#from thorn import model_reverser, webhook_model
```

Comment:
Style: I'd remove all of the commented out lines once you are sure it does what you want.

Reply:
Ooh good catch, I found a couple other thorn comments to remove as well.